MapzenSDK/Marker.swift

```diff
@@ -234,12 +245,20 @@ public class PointMarker : GeometricMarker, GenericPointIconMarker {
    - parameter ease: Easing to use for animation.
    */
   public func setPointEased(_ coordinates: TGGeoPoint, seconds: Float, easeType ease: TGEaseType) -> Bool {
-    return tgMarker.setPointEased(coordinates, seconds: seconds, easeType: ease)
+    tgMarker?.pointEased(coordinates, seconds: seconds, easeType: ease)
```

Comment:
~Do we need to call `getError` before `pointEased` to avoid returning an error for a previous action taken on `tgMarker`? I'm wondering if we have an existing error on the `tgMarker` and then call `pointEased` which completes without an error, will we erroneously return `false` here?~ I just saw @karimnaaji's comment, looks like you already plan to handle this